### PR TITLE
www-servers/spawn-fcgi: add 1.6.5

### PR DESCRIPTION
**Pull request by AndCycle**
version bump

---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by mattst88** [comment]
Is there any reason to disable IPv6 support in this package? We've been removing `IUSE=ipv6` when there's not a good reason to disable it.

**Comment by AndCycle** [comment]
> Is there any reason to disable IPv6 support in this package? We've been removing `IUSE=ipv6` when there's not a good reason to disable it.

I just copy the old ebuild for this version bump

https://redmine.lighttpd.net/projects/spawn-fcgi/repository/13/revisions/master/entry/NEWS
```
- 1.6.5 - 2022-12-21
  * Configure script cleanups (fixing detection of IPv6)
  * Update links and email addresses
```
there is an old bug related to ipv6 been fixed so think it's safe to remove `IUSE=ipv6`

**Comment by mattst88** [comment]
```
!!! Fetched file: spawn-fcgi-1.6.5.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      14369
!!! Expected: 14403
```

I don't know whether this is the result of the tarball being generated by github vs lighttpd's gitea.

I'd just get the file from http://download.lighttpd.net/spawn-fcgi/releases-1.6.x/spawn-fcgi-1.6.5.tar.gz which appears to be a static archive, and not generated on the fly.

Even better, there's an xz archive: http://download.lighttpd.net/spawn-fcgi/releases-1.6.x/spawn-fcgi-1.6.5.tar.xz

**Comment by mattst88** [comment]
These archives also have a `configure` script, so no need to `inherit autotools` or run `eautoreconf`.

File: www-servers/spawn-fcgi/spawn-fcgi-1.6.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="A FCGI spawner for lighttpd and cherokee and other webservers"
+HOMEPAGE="https://redmine.lighttpd.net/projects/spawn-fcgi/wiki"
+SRC_URI="
+	https://git.lighttpd.net/lighttpd/spawn-fcgi/archive/${P}.tar.gz
+	https://github.com/lighttpd/spawn-fcgi/archive/refs/tags/${P}.tar.gz
+"
```
Comment (mattst88) [inline www-servers/spawn-fcgi/spawn-fcgi-1.6.5.ebuild:13]: Switch to

```
SRC_URI="http://download.lighttpd.net/spawn-fcgi/releases-1.6.x/${P}.tar.xz"
```

File: www-servers/spawn-fcgi/spawn-fcgi-1.6.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+S="${WORKDIR}/${PN}"
```
Comment (mattst88) [inline www-servers/spawn-fcgi/spawn-fcgi-1.6.5.ebuild:14]: Can drop this with the new tarball.

File: www-servers/spawn-fcgi/spawn-fcgi-1.6.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+DEPEND=""
+RDEPEND=""
```
Comment (mattst88) [inline www-servers/spawn-fcgi/spawn-fcgi-1.6.5.ebuild:21]: Can drop these.

File: www-servers/spawn-fcgi/spawn-fcgi-1.6.5.ebuild [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+src_prepare() {
+	default
+	eautoreconf
+}
```
Comment (mattst88) [inline www-servers/spawn-fcgi/spawn-fcgi-1.6.5.ebuild:26]: Drop `src_prepare()`.

File: www-servers/spawn-fcgi/spawn-fcgi-1.6.5.ebuild [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
```
Comment (mattst88) [inline www-servers/spawn-fcgi/spawn-fcgi-1.6.5.ebuild:6]: Can drop this with the new tarball.

**Review by mattst88 — CHANGES_REQUESTED** [review]

**Comment by mattst88** [comment]
Can add a `remote-id` to github:

```
MissingRemoteId: missing <remote-id type="github">lighttpd/spawn-fcgi</remote-id> (inferred from URI 'https://github.com/lighttpd/spawn-fcgi/archive/refs/tags/spawn-fcgi-1.6.5.tar.gz')
```

**Comment by AndCycle** [comment]
thanks for the in depth review

**Pull request closed without merging**